lang/es.json

```diff
@@ -76,6 +77,7 @@
     "VTM5E.AddItem": "Añadir",
     "VTM5E.Edit": "Editar",
     "VTM5E.Delete": "Borrar",
+    "VTM5E.CustomRolls": "Custom Rolls",
```

Comment:
Can you find the right translation or leave it as VTM5E.CustomRolls? Makes it hard for me to check for missing localizations.